recipes/mosquitto/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "mosquitto" %}
+{% set version = "2.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://mosquitto.org/files/source/{{ name }}-{{ version }}.tar.gz
+  sha256: d0dde8fdb12caf6e2426b4f28081919a2fce3448773bdb8af0d3cd5fe5776925
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+
+  host:
+    - openssl
+    - libxslt
+
+
+test:
+  commands:
+    - mosquitto_sub --help || true  # [not win]
```

Comment:
Having `|| true` defeats the point of the test. I guess the help text exits with an error? If so can you do something like `mosquitto_sub --help | grep "something"` instead?

Reply:
I think this comment has been missed? There also needs to be a test for Windows if you're using `# [not win]`.